src/platform/site-wide/sass/merger.scss

```diff
@@ -187,13 +187,22 @@ $medium-phone-screen: 375px;
       @include media($medium-phone-screen) {
         span {
           max-width: 102px;
+
+          &.user-dropdown-email {
+            margin-top: 0;
+          }
         }
       }
 
       @include media($medium-screen) {
         span {
-          display: inline-block;
           padding-left: 0;
+          max-width: 300px;
+          height: 20px;
+
+          &.user-dropdown-email {
+            margin-top: 5px;
+          }
```

Comment:
It seems like we just changed the text based on screen size - how come we needed some new style rules?

Reply:
@ncksllvn because we are now using the mobile-responsive utility, it sets the `display: initial !important`. This causes alignment issues with the top user-nav area in medium-up. This compensate for that misalignment.